micrometer-tracing-tests/micrometer-tracing-test/src/main/java/io/micrometer/tracing/test/simple/SimpleBaggageManager.java

```diff
@@ -104,10 +106,21 @@ private Baggage createSimpleBaggage(String name) {
     }
 
     @Override
+    @Deprecated
```

Comment:
Could you please add a note to use `createBaggageInScope` instead?

Reply:
But this are inheriting from a method that already provides that info